packages/manager/src/features/Linodes/LinodesLanding/LinodeRow/LinodeRow.tsx

```diff
@@ -52,14 +55,18 @@ export const LinodeRow = (props: Props) => {
     status,
     type,
     maintenance,
-    vpcLabel,
-    vpcId,
   } = props;
 
   const notificationContext = React.useContext(_notificationContext);
 
   const { data: notifications } = useNotificationsQuery();
 
+  // TODO: VPC - later if there is a way to directly get a linode's vpc, replace this
+  const { data: configs } = useAllLinodeConfigsQuery(id);
+  const vpcId = getVPCId(configs ?? []);
+  const { data: vpc } = useVPCQuery(vpcId ?? -1);
```

Comment:
We should only "enable" this query if `vpcId` is defined. This will require you too add an `enabled` parameter and pass it to the useQuery for `useVPCQuery` 

```suggestion
  const { data: vpc } = useVPCQuery(vpcId ?? -1, vpcId !== undefined);
```

This will make sure we don't fire-off any VPC API calls until we have determined if the Linode has a VPC